packages/cardano-services/src/Utxo/DbSyncUtxoProvider/mappers.ts

```diff
@@ -0,0 +1,52 @@
+import { Cardano } from '@cardano-sdk/core';
+import { UtxoModel } from './types';
+
+const generateAssetId = (policy: string, name: string) =>
+  Cardano.AssetId(policy + (!name || name === '' ? '\\x' : name));
```

Comment:
Not sure if it is needed, I can delete it. But since we are representing names as hex strings I decided to put the hex representation of empty space which is `\\x`.